WTWSMS/common/religions/00_religions.txt

```diff
@@ -367,9 +367,11 @@ christian = {
 		can_grant_claim = yes
 		can_call_crusade = no
 		priests_can_inherit = no
-		max_wives = 4
+		max_wives = 1
```

Comment:
Why did you change the wives here? Why don't Insular Christians have three wives?